apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -198,9 +198,12 @@ export class ChangeAction extends BaseRushAction {
   private _hasProjectChanged(changedFolders: Array<string | undefined>,
     project: RushConfigurationProject): boolean {
     let normalizedFolder: string = project.projectRelativeFolder;
+
+    const rushPathDiff: string = this._findRushPathDiff(changedFolders, project);
     if (normalizedFolder.charAt(normalizedFolder.length - 1) !== '/') {
       normalizedFolder = normalizedFolder + '/';
     }
+    normalizedFolder = rushPathDiff + normalizedFolder;
```

Comment:
Use `path.join` to concatenate paths.

```suggestion
    normalizedFolder = path.join(rushPathDiff, normalizedFolder);
```